tests/tls/bug-7286-tls-metadata-02/test.yaml

```diff
@@ -0,0 +1,21 @@
+requires:
+    min-version: 7.0.6
```

Comment:
This looks good but sadly due to the json schema issue, the test won't work even on this min version..
Wondering if there should be an option to disable schema checks for tests like these..

Reply:
Maybe a PR template option, or label? Like, we still want the test to be complete, and hopefully update the schema as SV highlights an issue like this, but then at least we'd still be able to check whether the tests are testing what they're supposed to, in the meanwhile.

Should then maybe this one be merged first? https://github.com/OISF/suricata/pull/11845

Reply:
> Maybe a PR template option, or label? Like, we still want the test to be complete, and hopefully update the schema as SV highlights an issue like this, but then at least we'd still be able to check whether the tests are testing what they're supposed to, in the meanwhile.
> 

I think it'll have to be a test.yaml option as validation is run by s-v and not the CI checks.

> Should then maybe this one be merged first? [OISF/suricata#11845](https://github.com/OISF/suricata/pull/11845) (or at least the Schema fixing PR)

We should indeed merge the Suricata PRs and perhaps this one too.

Reply:
> > Maybe a PR template option, or label? Like, we still want the test to be complete, and hopefully update the schema as SV highlights an issue like this, but then at least we'd still be able to check whether the tests are testing what they're supposed to, in the meanwhile.
> 
> I think it'll have to be a test.yaml option as validation is run by s-v and not the CI checks.

Should I create a ticket?

Reply:
No, it's trivial really and does not need to be tracked. Creating PR